mirrord/layer/tests/apps/app_flask.py

```diff
@@ -6,7 +6,7 @@
 from os import getpid, kill
 from signal import SIGTERM
 
-from flask import Flask
+from flask import Flask, request
```

Comment:
```suggestion
from flask import Flask
```

Reply:
shouldn't have committed this to begin with, nice catch.